watertap/examples/chemistry/tests/test_pH_dependent_solubility.py

```diff
@@ -946,13 +946,16 @@ def run_case2(
     init_options = {**solver.options}
     init_options["tol"] = init_tol
     init_options["constr_viol_tol"] = init_tol
+    init_options["ma27_pivtol"] = 5e-1
```

Comment:
Have we tested this as a global default? Or does that break many flowsheets?

Reply:
I have not tested it extensively. I think I will explore it as a next step.

Ipopt will increase this automatically if it thinks it's "needed", so an in-between would be to set `ma27_pivtolmax` higher than the current default of `1e-04`.